hw/ip/keymgr/keymgr.core

```diff
@@ -15,6 +15,7 @@ filesets:
       - lowrisc:prim:msb_extend
       - lowrisc:ip:keymgr_pkg
       - lowrisc:ip:kmac_pkg
+      - lowrisc:ip:rom_ctrl
```

Comment:
Maybe we should split out a `rom_ctrl_pkg` to avoid `keymgr` needing to depend on the whole of `rom_ctrl`?

Reply:
ah yes, i would prefer that, but didn't want to disturb the existing file hierarchy.
I can do that if you prefer? not a problem for me.

Reply:
i'm going to merge this one for now (due to long ci turn-around times), and follow up with splitting the core file.s